run_brer/run_config.py

```diff
@@ -583,6 +583,8 @@ def run(self, tpr_file=None, **kwargs):
             else:
                 assert end_time > start_time
                 if end_time - start_time >= requested_production_time:
+                    prev_iter_state_json = 'state_' + str(self.run_data.get('iteration')) + '.json'
```

Comment:
I think we should put the copy in a predictable place with an absolute path. Does it seem reasonable to use the same directory as the regular state file?
```suggestion
                    state_dir = os.path.dirname(self.state_json)
                    archive_name = 'state_' + str(self.run_data.get('iteration')) + '.json'
                    prev_iter_state_json = os.path.join(state_dir, archive_name)
```

Reply:
Reasonable! I'll add this change